modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/AssaultCommand.java

```diff
@@ -115,16 +114,10 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
             return true;
         }
 //			if(aRegion.getType() instanceof ProtectedCuboidRegion) { // Originally I wasn't going to do non-cubes, but we'll try it and see how it goes. In theory it may repair more than it should but... meh...
-        String repairStateName = Movecraft.getInstance().getDataFolder().getAbsolutePath() + "/RegionRepairStates";
-        File file = new File(repairStateName);
-        if (!file.exists()) {
-            file.mkdirs();
-        }
-        repairStateName += "/";
-        repairStateName += aRegion.getId().replaceAll("\\s+", "_");
-        repairStateName += ".schematic";
-        file = new File(repairStateName);
-
+        if (!MovecraftRepair.getInstance().saveRegionRepairState(player.getWorld(), aRegion)){
```

Comment:
Did you switch to saving the entire region instead of a portion like before?

Reply:
That may have been outdated, but I believe yarro only allowed the closest 200x200 blocks of a region to be assaulted, the rest would still cause damages but nothing would blow up.

Reply:
it saves the entire region yes